### ISSUE TITLE
regarding jira integration with Hygieia

### ISSUE THREAD

**Issue by Navya-V** [Hygieia/hygieia-feature-jira-collector#25]
hi ...i am  not able to understand what is the custom_field (feature.jiraSprintDataFieldName=customfield_10000)  what is the value that we have to provide in the properties file,where do we find the custom_feild ....please somebody guide i get 0 0 0 on dashboard

**Comment by Navya-V** [comment]
got the info...it is visible on jira dashboard with admin privilages
thanks

**Comment by larsen-mallick** [comment]
Am still stuck with this. 

Can someone please say where I can identify the cutom fields?

**Comment by Navya-V** [comment]
> Am still stuck with this.
> 
> Can someone please say where I can identify the cutom fields?

just check the jira account with admin privilages , ull be able to find the custom_feilds and their ids

**Comment by larsen-mallick** [comment]
As recommend in Hygieia documentation, I got custom field values by hitting issueType endpoint. 

Though tempo-teams API was always returning 404, so I had to set - feature.jiraBoardAsTeam=true and get board information instead of teams.  Now, this has all started to work.

Still, out of curiosity, where in JIRA admin page I can find custom field details?

A screenshot will be much helpful. Thanks.
